src/bot/commands/random/trivia.ts

```diff
@@ -141,108 +153,207 @@ export class Trivia extends Command {
-                        {
-                            name: 'D)',
-                            value: answers[3]
-                        }],
-                    }
-                }) as Message;
-                await messageMultiple.react('🇦');
-                await messageMultiple.react('🇧');
-                await messageMultiple.react('🇨');
-                await messageMultiple.react('🇩');
+        // Add the reactions
+        if (multipleChoice) {
+            for (let i = 0; i < choiceEmoji.length; i++){
+                await message.react(choiceEmoji[i]);
             }
-            else {
-                let messageBoolean = await input.channel.send({
-                    embed:
-                    {
-                        color: 3447003,
-                        title: '__' + parsed.results[count].category + '__',
-                        description: question,
-                        fields: [{
-                            name: 'A)',
-                            value: answers[0]
-                        },
-                        {
-                            name: 'B)',
-                            value: answers[1]
-                        }],
-                    }
-                }) as Message;
-                await messageBoolean.react('🇦');
-                await messageBoolean.react('🇧');
+        }
+        else {
+            for (let i = 0; i < choiceEmoji.length/2; i++){
+                await message.react(choiceEmoji[i]);
             }
+        }
+
+        // Countdown (20 sec)
+        let countdownMessage = await input.channel.send(`${Emoji.LOADING}  15 seconds left...`) as Message;
+        let timer = new Timer(15, async function (remaining) {
+            await countdownMessage.edit(`${Emoji.LOADING}  ${remaining} seconds left...`);
+        });
+        timer.run();
+        await timer.wait();
 
-            count++;
-            //   const collector = (reaction, user)
-        //}
-        //while (count < 10)
+        // Close listener
+        listener.close();
+
+        // Split into two groups (correct and incorrect)
+        let correct: GuildMember[] = [];
+        let incorrect: GuildMember[] = [];
+        let correctNames : string[] = [];
+        let incorrectNames : string[] = [];
+
+        _.each(reactionAnswers, (wasCorrect, id) => {
+            // Get the member instance from their id
+            let member = input.guild.member(id);
+
+            // Add them to the appropriate array
+            if (wasCorrect){
+                correct.push(member);
+                correctNames.push(member.displayName);
+            } 
+            else{
+                incorrect.push(member);
+                incorrectNames.push(member.displayName);
+            } 
+        });
+        
+        //If no one answers delete the question
+        if (_.size(reactionAnswers) == 0) {
+            countdownMessage.edit(`${Emoji.LOADING} No answers chosen, shutting down...`);
```

Comment:
```suggestion
            countdownMessage.edit(`${Emoji.LOADING}  No answers chosen, shutting down...`);
```
The convention for our bot is two spaces after an emoji when using the emoji as an icon.